src/commands/upload_file.ts

```diff
@@ -134,12 +135,22 @@ async function getRemoteFile(
 	}
 
 	const tempFolder = getTempFolder();
-
-	const localFilePath = await download(remoteFilePath, tempFolder);
+	const destinationFileName = parameters.getRequiredParameterValue(DestinationFileNameParameter);
+
+	const { pathToFile, contentType } = await download(
+		remoteFilePath,
+		tempFolder,
+		destinationFileName,
+		(downloadProgress: number) => {
+			if (showProgressLog) {
+				process.stderr.write(`Downloading file... ${downloadProgress}\r`);
```

Comment:
```suggestion
				process.stderr.write(`Downloading file... ${downloadProgress.toFixed(1)}\r`);
```
This will keep it to 1 decimal place.